pixart.js

```diff
@@ -1 +1,24 @@
+$(document).ready(function(){
+  $("#set-color").click(function(event){
+    event.preventDefault()
+       $(".brush").css("background-color",$("input").val())
+  });
+});
 
+$(document).ready(function(){
```

Comment:
You can just use one $(document).ready and just put all of your code inside the one. This code works because the [event loop](https://www.youtube.com/watch?v=8aGhZQkoFbQ) is queuing each of these three functions to execute when the dom is constructed but we really just want one $(document).ready